### PR TITLE
LPDs 23077 and 23030 | master

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LPD-23027
https://liferay.atlassian.net/browse/LPD-23030

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/segment-editor/dynamic/inputs/components/attribute-conjunction-input/index.tsx [hunk 1]
```diff
@@ -115,7 +114,7 @@ const AttributeFilterConjunctionInput: React.FC<IAttributeFilterConjunctionInput
 		});
 	};
 
-	const {dataType} = getAttributeFromContext();
+	const {dataType}: any = getAttributeFromContext();
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/segment-editor/dynamic/inputs/components/attribute-conjunction-input/index.tsx:117]: Try something like:
```
const {dataType}: {dataType?: DataTypes} = ...
```

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/segment-editor/dynamic/inputs/components/attribute-conjunction-input/index.tsx (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/segment-editor/dynamic/inputs/components/attribute-conjunction-input/index.tsx]: done!

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:stop

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:stable

**Pull request closed without merging**